iroha/iroha.py

```diff
@@ -341,19 +341,26 @@ class IrohaGrpc(object):
     Possible implementation of gRPC transport to Iroha
     """
 
-    def __init__(self, address=None, timeout=None, secure=False):
+    def __init__(self, address=None, timeout=None, secure=False, max_message_length=5 * 1024**3):
         """
         Create Iroha gRPC client
         :param address: Iroha Torii address with port, example "127.0.0.1:50051"
         :param timeout: timeout for network I/O operations in seconds
         :param secure: enable grpc ssl channel
+        :param max_message_length: it is max message length in bytes for grpc, default 5Mb
         """
         self._address = address if address else '127.0.0.1:50051'
 
         if secure:
-            self._channel = grpc.secure_channel(self._address, grpc.ssl_channel_credentials())
+            self._channel = grpc.secure_channel(self._address, grpc.ssl_channel_credentials(), options=[
+                ('grpc.max_send_message_length', max_message_length),
+                ('grpc.max_receive_message_length', max_message_length),
+            ])
```

Comment:
```suggestion
            self._channel = grpc.secure_channel(self._address, grpc.ssl_channel_credentials(), **channel_kwargs)
```
to
- avoid MxN problem with options
- keep transient default behaviour